projects/meson/README.md

```diff
@@ -167,7 +167,23 @@ ninja -C build-gtk install
 
 ## Cross-compiling
 
-1. Create a fake windows root directory and download windows libraries from MINGW64.
+1. Install the cross-compiler MINGW, if you don't have it already:
+
+    For Debian/Ubuntu Linux:
+
+        apt-get install g++-mingw-w64 wine64-development
+
+    The name of the compiler that we want to use is `x86_64-w64-mingw32-g++-posix`.
+    * The prefix `x86_64-w64-` indicates that we want to compile for 64-bit windows, not 32-bit windows.
+    * The ending `-posix` indicates that we want to use UNIX (POSIX) threads, not Windows threads.
+
+    There is also a homebrew package `mingw-w64` on Mac, but these instructions have not been tested on Mac.
+
+        brew install mingw-w64 wine-stable
+
+    You might need to update the paths in the cross-file below.
+
+2. Create a fake windows root directory and download windows libraries from MINGW64.
```

Comment:
Could it be worth providing an example command showing how to do this?

Reply:
Do you mean, to update the paths?

Reply:
I did mean creating a fake windows root directory, but somehow I failed to notice that the command immediately below the line in question did exactly that! Approving now.